docker-entrypoint.sh

```diff
@@ -5,7 +5,7 @@ set -e
 
 # If the user is trying to run atlantis directly with some arguments, then
 # pass them to atlantis.
-if [ "$(echo "${1}" | cut -c1)" ]; then
+if [ "$(echo "${1}" | cut -c1)" = "-" ]; then
```

Comment:
Please run hadolint on the Dockerfile and add the output to the pr body

https://github.com/hadolint/hadolint

Reply:
@dpiddock please also indicate that you have run tests above

Reply:
I ran hadolint against the Dockerfile. No output was produced. This PR doesn't modify the file either.

Reply:
@dpiddock sorry my mistake, I meant `shellcheck`

https://github.com/koalaman/shellcheck

Reply:
I checked through the shellcheck website and I only see this which already existed before your PR

https://www.shellcheck.net/
https://www.shellcheck.net/wiki/SC1008

```
#!/usr/bin/dumb-init /bin/sh
^-- SC1008 (error): This shebang was unrecognized. ShellCheck only supports sh/bash/dash/ksh. Add a 'shell' directive to specify.
```